src/python/turicreate/toolkits/object_detector/_tf_model_architecture.py

```diff
@@ -21,6 +21,8 @@ def __init__(self, input_h, input_w, batch_size, output_size, init_weights, conf
         _tf.reset_default_graph()
 
         # Suppresses verbosity to only errors
+        import os
```

Comment:
Can we make this into a utility and call it everywhere.

Reply:
Yeah that makes sense.